presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -438,6 +445,12 @@ public static Optional<BiFunction<Integer, URI, Process>> getExternalWorkerLaunc
                                 "system-memory-gb=4%n" +
                                 "http-server.http.port=%d", discoveryUri, port);
 
+                        if (sidecarEnabled.get()) {
```

Comment:
Can you help me understand why we are doing this?

Reply:
Removed these changes.